packages/babel-core/src/errors/rewrite-stack-trace.ts

```diff
@@ -131,7 +122,17 @@ function setupPrepareStackTrace() {
 
   const { prepareStackTrace = defaultPrepareStackTrace } = Error;
 
-  Error.stackTraceLimit += STACK_TRACE_LIMIT_DELTA;
+  // We add some extra frames to Error.stackTraceLimit, so that we can
+  // always show some useful frames even after deleting ours.
+  // STACK_TRACE_LIMIT_DELTA should be around the maximum expected number
+  // of internal frames, and not too big because capturing the stack trace
+  // is slow (this is why Error.stackTraceLimit does not default to Infinity!).
+  // Increase it if needed.
+  const MIN_STACK_TRACE_LIMIT = 50;
+  Error.stackTraceLimit = Math.max(
```

Comment:
```js
  if (
    Error.stackTraceLimit < MIN_STACK_TRACE_LIMIT &&
    Error.stackTraceLimit > 10
  ) {
    Error.stackTraceLimit = MIN_STACK_TRACE_LIMIT;
  }
```
Would this be better?

Reply:
I only skipped the 0 case. If `Error.stackTraceLimit` is a low number, we would otherwise often make it 0 by deleting our own frames (and 0 is worse then "a few more than expected", when you still want _something_).